legend-engine-xt-nonrelationalStore-mongodb-grammar-integration/pom.xml

```diff
@@ -122,6 +122,11 @@
             <groupId>org.finos.legend.pure</groupId>
             <artifactId>legend-pure-m4</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-runtime-java-extension-dsl-mapping</artifactId>
+            <scope>runtime</scope>
```

Comment:
if I don't have this dependency, I get used undeclared dependency error, if I add it I get unused declared dependency